docs/pages/versions/unversioned/sdk/location.md

```diff
@@ -406,6 +412,13 @@ Object of type `Region` includes following fields:
 | `GeofencingRegionState.Inside`  | 1     | Indicates that the device is inside the region. |
 | `GeofencingRegionState.Outside` | 2     | Inverse of inside state.                        |
 
+### `Location.MethodType`
+
+| Method type                           | Value | Description                                      |
+| --------------------------------------| ----- | ------------------------------------------------ |
+| `Location.MethodType.MethodType.Slow` | 1     | Use slower method to calculate current position. |
+| `Location.MethodType.MethodType.Fast` | 2     | Use faster method to calculate current position. |
```

Comment:
`Location.MethodType.MethodType.{Slow, Fast}`?
Shouldn't that be `Location.MethodType.{Slow, Fast}`?